matrix_client/client.py

```diff
@@ -144,6 +150,9 @@ def __init__(self, base_url, token=None, user_id=None,
         self.rooms = {
             # room_id: Room
         }
+        self.users = {
```

Comment:
Sure. Shouldn't `Client.rooms` be the same?

Reply:
In fact, not 100% sure how you want class attributes to be read-only (maybe only an indication in docstring would have been enough?). I implemented this with a `property`, but to me it fixes half of the problem since the internal dict can still be modified. I'd prefer returning a copy, but a copy is expensive.

Reply:
Settled for docstring note + `property` then.

Reply:
tbh, with still mutable dicts, I don't think changing it to a "property" really gains us much. :/ Definitely thanks for the docstring update though.

Reply:
Absolutely agree, just wasn't sure what you expected. Should be good now!